ArduPlane/mode_qland.cpp

```diff
@@ -9,7 +9,12 @@ bool ModeQLand::_enter()
     quadplane.throttle_wait = false;
     quadplane.setup_target_position();
     poscontrol.set_state(QuadPlane::QPOS_LAND_DESCEND);
+
+    // Clear any pilot corrections
```

Comment:
There could be a argument to do this on every mode change, not just into QLand.

Reply:
i think every mode change makes sense

Reply:
Now on every mode change.